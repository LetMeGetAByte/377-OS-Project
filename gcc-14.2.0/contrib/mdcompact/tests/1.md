version https://git-lfs.github.com/spec/v1
oid sha256:e3fb1711a55dfa0359fbdc468337325f7f6527d7ba3c5f8f86ebab4006b122ce
size 1316
