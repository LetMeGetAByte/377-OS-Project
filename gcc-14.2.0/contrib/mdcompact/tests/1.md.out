version https://git-lfs.github.com/spec/v1
oid sha256:be72c903d43e0194b3bdb8567937e421d584d6546b36cc0935c949b723f6507c
size 1817
